### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by megan07**
3.38.0 Changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+* **New Resource:** google_data_loss_prevention_inspect_template ([#7123](https://github.com/hashicorp/terraform-provider-google/pull/7123))
+* **New Resource:** google_data_loss_prevention_job_trigger ([#7123](https://github.com/hashicorp/terraform-provider-google/pull/7123))
+* **New Resource:** google_data_loss_prevention_stored_info_type ([#7145](https://github.com/hashicorp/terraform-provider-google/pull/7145))
+* **New Resource:** google_project_service_identity ([#7133](https://github.com/hashicorp/terraform-provider-google/pull/7133))
+
+IMPROVEMENTS:
+* compute: Added graceful termination to `google_compute_instance_group_manager` create calls so that partially created instance group managers will resume the original operation if the Terraform process is killed mid create. ([#7153](https://github.com/hashicorp/terraform-provider-google/pull/7153))
+* container: added project override support to `google_container_cluster` and `google_container_nodepool` ([#7114](https://github.com/hashicorp/terraform-provider-google/pull/7114))
+* google_project_service_identity: added `email` output. ([#7162](https://github.com/hashicorp/terraform-provider-google/pull/7162))
```
Comment (rileykarson) [inline CHANGELOG.md:20]: Since the resource is released in this CL, this can be omitted.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+* osconfig: added rollout field to `google_os_config_patch_deployment` ([#7172](https://github.com/hashicorp/terraform-provider-google/pull/7172))
+* provider: added a new field `billing_project` to the provider that's associated as a billing/quota project with most requests when `user_project_override` true ([#7113](https://github.com/hashicorp/terraform-provider-google/pull/7113))
```
Comment (rileykarson) [inline CHANGELOG.md:22]:
```suggestion
* provider: added a new field `billing_project` to the provider that's associated as a billing/quota project with most requests when `user_project_override` is true ([#7113](https://github.com/hashicorp/terraform-provider-google/pull/7113))
```

Whoops, omission on my part

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
+* resource-manager: added additional fields to `google_projects` datasource ([#7139](https://github.com/hashicorp/terraform-provider-google/pull/7139))
```
Comment (rileykarson) [inline CHANGELOG.md:23]:
```suggestion
* resourcemanager: added additional fields to `google_projects` datasource ([#7139](https://github.com/hashicorp/terraform-provider-google/pull/7139))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,33 @@
...
+* **New Resource:** google_compute_disk_iam_policy ([#7110](https://github.com/hashicorp/terraform-provider-google/pull/7110))
+* **New Resource:** google_compute_region_disk_iam_binding ([#7110](https://github.com/hashicorp/terraform-provider-google/pull/7110))
+* **New Resource:** google_compute_region_disk_iam_member ([#7110](https://github.com/hashicorp/terraform-provider-google/pull/7110))
+* **New Resource:** google_compute_region_disk_iam_policy ([#7110](https://github.com/hashicorp/terraform-provider-google/pull/7110))
+* **New Resource:** google_data_loss_prevention_inspect_template ([#7123](https://github.com/hashicorp/terraform-provider-google/pull/7123))
+* **New Resource:** google_data_loss_prevention_job_trigger ([#7123](https://github.com/hashicorp/terraform-provider-google/pull/7123))
+* **New Resource:** google_data_loss_prevention_stored_info_type ([#7145](https://github.com/hashicorp/terraform-provider-google/pull/7145))
+* **New Resource:** google_project_service_identity ([#7133](https://github.com/hashicorp/terraform-provider-google/pull/7133))
```
Comment (danawillow) [inline CHANGELOG.md:15]: this is beta-only

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,33 @@
...
+* container: added project override support to `google_container_cluster` and `google_container_nodepool` ([#7114](https://github.com/hashicorp/terraform-provider-google/pull/7114))
+* google_project_service_identity: added `email` output. ([#7162](https://github.com/hashicorp/terraform-provider-google/pull/7162))
+* osconfig: added rollout field to `google_os_config_patch_deployment` ([#7172](https://github.com/hashicorp/terraform-provider-google/pull/7172))
+* provider: added a new field `billing_project` to the provider that's associated as a billing/quota project with most requests when `user_project_override` true ([#7113](https://github.com/hashicorp/terraform-provider-google/pull/7113))
+* resource-manager: added additional fields to `google_projects` datasource ([#7139](https://github.com/hashicorp/terraform-provider-google/pull/7139))
+* servicemanagement: added project override support to `google_project_service` ([#7114](https://github.com/hashicorp/terraform-provider-google/pull/7114))
```
Comment (danawillow) [inline CHANGELOG.md:24]:
```suggestion
* serviceusage: added project override support to `google_project_service` ([#7114](https://github.com/hashicorp/terraform-provider-google/pull/7114))
```

**Pull request closed without merging**